Packs/CortexXDR/Scripts/XDRConnectedEndpoints/XDRConnectedEndpoints.py

```diff
@@ -0,0 +1,39 @@
+import traceback
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def main():
+    try:
+        args = demisto.args()
+        last_seen_gte = args.get('from')
+        last_seen_lte = args.get('to')
```

Comment:
I think that more descriptive variable names will help understand what is the meaning of them.
Currently, it's difficult to understand.